lib/helpers.js

```diff
@@ -9,6 +9,21 @@ const logger = app.locals.logger;
 /**
  * Helper functions.
  */
+
+/**
+ * Returns whether given Campaign id is enabled for CampaignBot.
+ */
+module.exports.isCampaignBotCampaign = function (campaignId) {
+  // TODO: We'll eventually determine if a CampaignBotCampaign is enabled if a keyword exists for
+  // its id (and its campaign.status is active).
+  const campaignDoc = app.locals.campaigns[campaignId];
```

Comment:
Wanted to check for the `_id` specifically to make sure whatever is stored in the array is a loaded document.. but maybe it's not necessary?